extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -57,7 +57,8 @@ class AmpAccordion extends AMP.BaseElement {
         content.setAttribute('id', contentId);
       }
       header.setAttribute('aria-controls', contentId);
-      header.addEventListener('click', this.onHeaderClick_.bind(this));
+      const boundOnHeaderClick_ = this.onHeaderClick_.bind(this);
```

Comment:
We'll need to hoist this outside the `#forEach`, else we'll still create X bound event listeners.